src/main.ts

```diff
@@ -1,10 +1,12 @@
-
```

Comment:
The changes improve code readability by adding comments. However, there's a minor improvement opportunity.

``` 
import { createApp } from 'vue'
import App from './App.vue'
import router from './router'

const app = createApp(App)

app.use(router)

app.mount('#app')
```